packages/user/src/services/user-module.ts

```diff
@@ -103,11 +114,60 @@ export default class UserModuleService<TUser extends User = User>
     return Array.isArray(data) ? serializedUsers : serializedUsers[0]
   }
 
+  createInvites(
+    data: UserTypes.CreateInviteDTO[],
+    sharedContext?: Context
+  ): Promise<UserTypes.InviteDTO[]>
+  createInvites(
+    data: UserTypes.CreateInviteDTO,
+    sharedContext?: Context
+  ): Promise<UserTypes.InviteDTO>
+
+  @InjectTransactionManager("baseRepository_")
+  async createInvites(
+    data: UserTypes.CreateInviteDTO[] | UserTypes.CreateInviteDTO,
+    @MedusaContext() sharedContext: Context = {}
+  ): Promise<UserTypes.InviteDTO | UserTypes.InviteDTO[]> {
+    const input = Array.isArray(data) ? data : [data]
+
+    const invites = await this.inviteService_.create(input, sharedContext)
+
+    const serializedInvites = await this.baseRepository_.serialize<
+      UserTypes.InviteDTO[]
+    >(invites, {
+      populate: true,
+    })
+
+    return serializedInvites
```

Comment:
todo: missing Array.isArray etc

Reply:
That is handled in the create method of the abstract internal service factory